css/style.css

```diff
@@ -16,4 +10,209 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* General formating */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-size: 16px;
+}
+
+html,
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  color: #000;
+  text-align: center;
+  background-image: url("img/first-background.jpg");
+  margin: 0;
+  font-size: 16px;
+  line-height: 1.6;
+}
+
+/* Header section */
+.logo {
+  max-width: 1.625rem;
+}
+
+.nav-bar {
+  background-color: #fff;
+  width: 100%;
+  height: 70px;
+  margin-top: 0;
+  position: fixed;
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  z-index: 1;
+  border-bottom: 1px solid #eaebec;
+}
+
+.nav-bar ul li a {
+  text-decoration: none;
+  color: #000;
+  opacity: 0.5;
+}
+
+.nav-bar ul li a:hover {
+  opacity: 1;
+}
+
+/* Lists and list items */
+ul {
+  list-style-type: none;
+  display: flex;
+  justify-content: center;
+}
+
+ul li {
+  padding: 1rem;
+}
+
+.bold {
+  font-weight: bolder;
+}
+
+/* First section */
+#hero {
+  background-image: url("../img/first-background.jpg");
+  background-size: auto;
+  background-position: center;
+  height: 112vh;
```

Comment:
Why 112vh? 100vh would work well :)

Also watch out, you'll need `background-repeat: no-repeat` for the image to show only once.

Reply:
100vh is not displaying as much of the image as in your original page for some reason..